lib/Models/Catalog/CatalogItems/MapboxVectorTileCatalogItem.ts

```diff
@@ -146,8 +146,7 @@ class MapboxVectorTileCatalogItem extends MappableMixin(
     }
   }
 
-  @computed
-  /** Convert traits into paint rules:
+  @computed /** Convert traits into paint rules:
```

Comment:
Prettier 3.4.2 also has this problem. Split out the change that moves the JSDoc to #7440 so that can get merged now without having to update the formatter.